pyproject.toml

```diff
@@ -29,7 +29,7 @@ classifiers = [
 ]
 requires-python = ">=3.9,<3.13"
 dependencies = [
-    'easyscience @ git+https://github.com/EasyScience/EasyScience.git@develop',
+    'easyscience @ git+https://github.com/EasyScience/EasyScience.git@unique_name_bug',
```

Comment:
Can be changed back to develop once the pull request has been merged :) 
putting this only as a reminder

Reply:
done